tools/test-proxy/Azure.Sdk.Tools.TestProxy/ConsoleWrapper/ConsoleWrapperTester.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace Azure.Sdk.Tools.TestProxy.ConsoleWrapper
```

Comment:
@JimSuplizio @scbedd If it's useful/interesting, I built a similar console wrapper thing to test an interactive CLI tool I wrote: https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/kubernetes/generator/Stress.Generator.Tests/GeneratorTest.cs#L8. It was made specifically to handle more complex branched input sessions by allowing you to set a queue of mocked inputs/responses/expectations.

Reply:
@benbp I went brain dead simple because there's one command that requires user input which is literally Y|N. If this expands I'll look into something like you have (or I'll just 'borrow' it).

Reply:
@JimSuplizio yeah. We'll want to copy that code out of stress into common libraries. I'm betting the same thing will happen with Test-Proxy and our `GitProcessHandler`.

Reply:
Yep, I agree keep it simple but if the inputs get more complex feel free to steal the code and/or refactor into common.